config/api/src/main/java/org/jboss/windup/config/AbstractPathConfigurationOption.java

```diff
@@ -65,12 +66,14 @@ else if (!Files.exists(path))
         return ValidationResult.SUCCESS;
     }
 
+    @SuppressWarnings("rawtypes")
     @Override
     public ValidationResult validate(Object fileObject)
     {
-        if (fileObject == null && isRequired())
+        if ( (fileObject == null && isRequired()) ||
+             (fileObject instanceof Collection && ((Collection) fileObject).isEmpty()) )
         {
-            return new ValidationResult(ValidationResult.Level.ERROR, getName() + " is required!");
+            return new ValidationResult(ValidationResult.Level.ERROR, getLabel() + " must be specified.");
```

Comment:
I think better to use label(s) in ValidationResult messages instead of name which is short.